filereader/texttemplate/texttemplate.go

```diff
@@ -21,9 +22,27 @@ func Parse(filename string, funcs template.FuncMap) (*template.Template, error)
 			}
 			return content, nil
 		},
+		"toJSON": func(v interface{}) (string, error) {
```

Comment:
I didn't check if it exists in  a code, because it is undocumented on their godoc page: https://godoc.org/github.com/Masterminds/sprig
 
so I am not sure we can rely on it being available all the time